cypress/integration/inventory/actions-menu.spec.js

```diff
@@ -1,23 +1,36 @@
 import TopMenu from '../../support/fragments/topMenu';
 import Actions from '../../support/fragments/inventory/actions';
+import InventorySearchPanel from '../../support/fragments/inventory/inventorySearchPanel';
 
 describe('ui-inventory: actions', () => {
-  beforeEach('navigates to Actions', () => {
+  beforeEach('navigates to actions', () => {
     cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
     cy.visit(TopMenu.inventoryPath);
   });
 
   it('C196752 verifies action menu options before any search is conducted', () => {
-    const optionsShouldDisabled = [
-      Actions.saveUUIDOption,
-      Actions.saveCQLQueryOption,
-      Actions.exportMARCOption,
-      Actions.showSelectedRecordsOption
-    ];
+    cy.do(Actions.open());
 
-    Actions.open()
-      .then(() => {
-        optionsShouldDisabled.forEach(option => Actions.optionIsDisabled(option, true));
-      });
+    cy.expect([
+      Actions.saveUUIDsOption().is({ disabled: true }),
```

Comment:
As I see test related with initial state of Inventories list. Need to add assertion on void list of inventories. May be there are some additional cases, for example, when inventories presented but not selected(please, review this point once again please, I don't know all specifics)

Reply:
If inventories presented but not selected It means something went wrong and test failed. It is ok.

Reply:
as I see in test precondition: "and the results pane is empty.". I can't see evident validation of this requirement in your test. Such additional check can get you answer about the reason of failed test quicker. It is one more benefit of my suggestion.